.gitignore

```diff
@@ -6,4 +6,6 @@ nbproject
 *.iml
 build
 /out
-**/*.gpg
\ No newline at end of file
+**/*.gpg
+keanu-python/dist/
```

Comment:
`keanu-python` has its own `.gitignore` file

Reply:
good call - I'll change it